### PR TITLE
[TASK] Load mode style inline

### PR DESCRIPTION
**Pull request by xf-**
Why?
Inline overhead is minimal and currently only known mode is night mode. Maybe a April fools day can be added, but in general to switch complete style is normally not the use case. Also link implementation still works, but the label color issue is back (tries to read the label-color from css, but value isn't updated/loaded).

Usage:
```html
 <link rel="stylesheet" class="css-mode night" media="not" href="night.css"> 
```
or
```html
<style class="css-mode night" media="not">
   <inline src="night.css" />
</style>
```
Advantage(s):
- Less JS code, setTimeout loop and dynamic link tags.
- No visible color switch is gone.

Disadvantage:
Night style is shipped by default (even without any mode).

### CODE REVIEW

File: lib/map.js [hunk 1]
```diff
@@ -330,25 +330,15 @@ define(['map/clientlayer', 'map/labelslayer',
         if (map.getZoom() > map.options.maxZoom) {
           map.setZoom(map.options.maxZoom);
         }
-        if (helper.localStorageTest()) {
-          localStorage.setItem('map/selectedLayer', JSON.stringify({ name: e.name }));
-        }
```
Comment (genofire) [inline lib/map.js:335]: Wo kamm das denn mal her xD
(Das war doch mal ein versehen bzw. nicht entfernter Code oder?)

**Review by genofire — APPROVED** [review]
Its good, the shipping of the night themen is okay - maybe we need a documentation "how to add and remove mode's by layer and there css"

File: lib/map.js (hunk 1 above)
Comment (xf-) [reply lib/map.js:335]: ich vermute das war ein ueberreset wie ganze layerstack immer ausgeliefert wurde.

**Comment by xf-** [comment]
@genofire README.md updated

**Comment by genofire** [comment]
I will merge, if i would have the permissions :D

File: tasks/build.js [hunk 2]
```diff
@@ -42,24 +42,37 @@ module.exports = function exports(grunt) {
...
       dev: {
         options: {
           sourceMap: true,
           outputStyle: 'expanded'
         },
-        files: {
-          'build/style.css': 'scss/main.scss',
-          'build/night.css': 'scss/night.scss'
-        }
+        files: [{
```
Comment (genofire) [inline tasks/build.js:57]: Top :)

**Review by genofire — APPROVED** [review]

**Pull request merged**